packages/ui/src/components/button.tsx

```diff
@@ -77,7 +77,8 @@ const buttonVariants = cva('cn-button', {
   ],
   defaultVariants: {
     variant: 'primary',
-    size: 'default'
+    size: 'default',
+    theme: 'default'
```

Comment:
Docs for button has been updated for this change already. Not sure why it has been removed.